src/mbgl/annotation/annotation_source.cpp

```diff
@@ -26,4 +30,8 @@ bool AnnotationSource::supportsLayerType(const mbgl::style::LayerTypeInfo* info)
     return !std::strcmp(info->type, "line") || !std::strcmp(info->type, "symbol") || !std::strcmp(info->type, "fill");
 }
 
+Mutable<Source::Impl> AnnotationSource::createMutable() const noexcept {
+    return staticMutableCast<Source::Impl>(makeMutable<Impl>(impl()));
```

Comment:
does it need explicit `staticMutableCast<Source::Impl>` ?

Reply:
Yes, it needs :disappointed: